l10n.yaml

```
@@ -0,0 +1,6 @@
+arb-dir: lib/l10n
+template-arb-file: app_en.arb
```

From reading through those docs on this config file, one other option looks like it'd be good to add:
```suggestion
template-arb-file: app_en.arb
required-resource-attributes: true
```

> Requires all resource ids to contain a corresponding resource attribute.
>
> By default, simple messages won’t require metadata, but it’s highly recommended as this provides context for the meaning of a message to readers.

We can always relent on that if we decide it's a pain. But my guess is that it won't actually be much of a burden to write a few words about what the message is for, at the time when we're adding it and it's right in front of us. (And I see you've consistently done that in the demo messages in this PR.)